app/konnect/dev-portal/index.md

```diff
@@ -40,8 +40,26 @@ For developers:
 
 * [Generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/) - This guide walks you through the process of generating and deleting API keys on the Dev Portal platform.
 
+### Contextual developer analytics 
+
+You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable dashboard. 
+
+Each application has its own dashboard, which provides a high-level summary of the **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
+
+* Requests by version
+* P99 latency by version
+* Error code distribution 
+
+All of these metrics can be configured across a time range up to **30 days**. 
```

Comment:
Relative timeframe is up to "Last 30 days" but you can go back up to 90 days with custom.